app/enterprise/1.5.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -0,0 +1,179 @@
+  multiple controller instances, setting the `CONTROLLER_KONG_WORKSPACE`
+  environment variable to the workspace that instance should use. These
+  instances should set `CONTROLLER_INGRESS_CLASS` to unique values, to avoid
+  creating duplicate configuration in workspaces. Note that if controller
+  instances are deployed outside the Kong pod the Admin API must be exposed,
+  and users should enable RBAC with workspace admin users for the controllers.
+  Set `CONTROLLER_KONG_ADMIN_TOKEN` to the RBAC user's token.
+* The controller cannot manage configuration for the features above: it cannot
+  create workspaces, Dev Portal content, admins, etc. These features must be
+  configured manually through the Admin API.
+
+### Plugin compatibility
+
+Not all plugins are compatible with DB-less operation, and as such not all
+plugins are available in the `kong-enterprise-k8s` image. In addition to the
+[compatible core plugins][core-plugins], the `kong-enterprise-k8s` image
+includes the following Enterprise plugins:
+
+* canary release
+* collector
+* degraphql
+* forward-proxy
+* graphql-proxy-cache-advanced
+* graphql-rate-limiting-advanced
+* jwt-signer
+* kafka-log
+* kafka-upstream
+* ldap-auth-advanced
+* mtls-auth
+* oauth2-introspection
+* openid-connect
+* proxy-cache-advanced
+* rate-limiting-advanced
+* request-validator
+* response-transformer-advanced
+* vault-auth
+
+The following Enterprise plugins are not included:
+
+* application-registration (only used with Dev Portal)
+* exit-transformer
+* key-auth-enc
+* request-transformer-advanced (request-transformer now has the same feature
+  set)
+* route-transformer-advanced
+* statsd-advanced (only used with Vitals)
+
+Third-party plugins are generally compatible with DB-less as long as they do
+not create custom entities (i.e. they do not add new entities that users can
+create and modify through the Admin API).
+
+### Manual configuration
+
+DB-less configuration must be supplied as a complete unit: it is not possible
+to add or modify entities individually through the Admin API, or provide
+partial configuration that is added to existing configuration. As such, all
+configuration must be sourced from Kubernetes resources so that the ingress
+controller can render it into a complete configuration.
+
+On database-backed deployments, users can create or modify configuration
+through the Admin API. The ingress controller uses a tag (set by the
+`CONTROLLER_KONG_ADMIN_FILTER_TAG` environment variable) to to identify
+configuration that it manages. While the controller will revert changes to
+configuration with its tag, other configuration is left as-is.
+
+Although database-backed deployments can use controller-generated and
+manually-added configuration simultaneously, Kong's recommended best practice
+is to still manage as much configuration through Kubernetes resources as
+possible, primarily using manual configuration for temporary or test resources.
```

Comment:
Let's explain first why controller-generated is the way to go, what problems can happen if you also do manually added configuration, and then let's explain how you can do manual configuration in a way that avoids conflicts.

This probably requires rewording this and the two paragraphs before it.

Reply:
Reworded last and added conflict avoidance strategies (though I'm somewhat averse to this since it tacitly implies that it's fine to mix the two), but left the order the same. I think it's important to establish the groundwork of what the controller is doing to understand how they coexist.